W15_lab06_elswenson_andrewtran1995.md

```diff
@@ -0,0 +1,9 @@
+cs56-W15-lab06
```

Comment:
Looks good!  Though you should rename this file to W15_lab06.md, rather than W15_lab06_elswenson_andrewtran1995.md